com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKRayReticleVisual.cs

```diff
@@ -0,0 +1,160 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// The reticle visual for a ray interactor. This behavior takes care of
+    /// aligning the reticle with a surface hit by the ray interactor.
+    /// </summary>
+    [AddComponentMenu("MRTK/Input/MRTK Ray Reticle Visual")]
+    public class MRTKRayReticleVisual : BaseReticleVisual
+    {
+        [SerializeField]
+        [Tooltip("The interactor which this visual represents.")]
+        private XRRayInteractor rayInteractor;
+
+        [SerializeField]
+        [Tooltip("The GameObject which holds the proximity light for the reticle")]
+        private GameObject proximityLight;
```

Comment:
Nit, extra lines, and separate serialized/nonserialized private fields (#regions would be even nicer, but not really necessary)

Reply:
Fixed